attestation.md

```diff
@@ -1,29 +1,33 @@
 # 🔖 Attestation 
 
 In this section, you'll play the role of the <span class="label-role attester">attester</span>:
-* You'll take a `RequestForAttestation` object; 
+
+* You'll take a `RequestForAttestation` object;
 * Attest it;
-* Store the attestation on the chain;
-* Build the `AttestedClaim` object, which will be send back to the <span class="label-role claimer">claimer</span>.
+* Store the attestation on the chain (more specifically only its hash, we'll get to that);
+* Build the `AttestedClaim` object which will be send back to the <span class="label-role claimer">claimer</span>.
+
+## Request KILT tokens
 
-## Get some tokens as an attester  
-When writing the hash of credentials on the blockchain, <span class="label-role attester">attesters</span> have to pay the angel’s
+When writing the hash of attestations on the blockchain, <span class="label-role attester">attesters</span> have to pay the angel’s
 share (gas or transaction fee) in KILT Tokens. So you'll need tokens to attest a claim.   
 
-Go to https://faucet.kilt.io/ and request tokens for your `<ATTESTER'S ADDRESS>`.
+Go to the [faucet] and request tokens for your `<ATTESTER'S ADDRESS>`.
```

Comment:
I tried connecting to the faucet but was unable to. `Cannot GET /[https://faucet.kilt.io/]`

Reply:
Thanks, fixed 👍